test-data/unit/stubgen.test

```diff
@@ -552,14 +602,148 @@ def f(a): ...
 [case testInferOptionalOnlyFunc]
 class A:
     x = None
-    def __init__(self, a=None) -> None:
```

Comment:
I would also keep an additional test for ``__init__``.

Reply:
Added